README.md

```diff
@@ -38,11 +38,16 @@ to the RDO project.
 * Follow the steps in [I want to add an easy fix issue](#i-want-to-add-an-easyfix-issue)
 
 ## Checklist for sending your first RDO gerrit Review:
-* [ ] Add the changed files using git add command
-* [ ] In the commit message: 
+* [ ] Add the changed files using **git add <modified files>** command
+* [ ] Then, run **git commit** command, It will open a editr to write commit message.
```

Comment:
This wasn't fixed, it also comes out with weird formatting, I think you need another line break - see here: https://github.com/chkumar246/easyfix/blob/d2a04baaa025b6a9a3fea6f554e9d8b707a5ea1c/README.md#checklist-for-sending-your-first-rdo-gerrit-review